services/connector/api.go

```diff
@@ -109,6 +109,7 @@ func (api *API) CallRPC(inputJSON string) (interface{}, error) {
 }
 
 func (api *API) RecallDAppPermission(origin string) error {
+	// TODO: close the websocket connection
```

Comment:
What if there's another app connected to the browser connect?

Related questions (for me to better understand 😄):
1. How can we manage the connection and close it?
2. Shouldn't the connection be always active? Closing it won't stop any further data transmission between plugin and Status?

Reply:
1. That is the main issue currently. I know of two ways to fix this:
- Make changes to the go-ethereum code and add the ability to control connections using signals or something else. That's a very bad idea
- Add a new separate websocket server directly to the connector service. I think this is the right way to do it, at the same time we can remove nesting and escaping in json. 

2. Websocket server works in one-to-many way, allowing several clients to access in one time. To close the connection you need to select a connection with origin matching to dApp's url.

Reply:
> Websocket server works in one-to-many way, allowing several clients to access in one time. To close the connection you need to select a connection with origin matching to dApp's url.

Does this mean the plugin will span a new connection for each dApp? I was assuming each plugin instance will establish a single connection for multiple dapps, acting like a proxy

Reply:
Yes, for each DApp there is a websocket handler. Here is the pictures for the current approach and what i advise to do
<img width="1181" alt="Screenshot 2024-08-15 at 17 50 20" src="https://github.com/user-attachments/assets/d68deea1-5ba5-4060-818e-7501688fc8a5">